docs/reference/package-control/repository.md

```diff
@@ -0,0 +1,779 @@
+	"releases": [
+		{
+			"asset": "*.sublime-package"
+		}
+	]
+}
+```
+
+Or `${version}`, `${st_build}` and `${platform}` can be used to match assets.
+Note that "platforms" must explicitly list each supported platform,
+for the platform variable to work.
+
+- `${platform}` — a platform-arch string as given in "platforms" list.
+  A separate explicit release is evaluated for each platform.
+  If `"platforms": ['*']` is specified, the variable is set to "any".
+- `${st_build}` — value of "sublime_text" stripped by leading operator:
+  - `*` — any
+  - `>=4107` — 4107
+  - `<4107` — 4107
+  - `4107 - 4126` — 4107
+- `${version}` — resolved semver without tag prefix: st4107-1.0.5 becomes 1.0.5
+
+```json
+{
+	"packages": [
+		{
+			"details": "https://github.com/SublimeText/Less",
+			"releases": [
+				{
+					"asset": "Less-${version}-st${st_build}.sublime-package",
+					"sublime_text": "4107 - 4148"
+				},
+				{
+					"asset": "Less-${version}-st${st_build}.sublime-package",
+					"sublime_text": ">=4149"
+				}
+			]
+		},
+		{
+			"details": "https://github.com/SublimeText/Less",
+			"releases": [
+				{
+					"asset": "Less-*.*.*-st4107.sublime-package",
+					"sublime_text": "4107 - 4148"
+				},
+				{
+					"asset": "Less-*.*.*-st4149.sublime-package",
+					"sublime_text": ">=4149"
+				}
+			]
+		},
+		{
+			"details": "https://github.com/SublimeText/PackageWithAsset",
+			"releases": [
+				{
+					"asset": "FileName-${platform}.sublime-package",
+					"platforms": ["linux-arm64", "linux-x64", "osx-arm64", "osx-x64", "windows-x64"]
+				}
+			]
+		},
+	]
+}
+```
+
+If names of download assets are not covered by the provided variable expansion,
+an explicit release specification can be declared for each platform.
+
+```json
+{
+	"details": "https://github.com/SublimeText/PackageWithAsset",
+	"releases": [
+		{
+			"asset": "FileName-linux-aarch64.sublime-package",
+			"platforms": ["linux-arm64"]
+		},
+		{
+			"asset": "FileName-linux-amd64.sublime-package",
+			"platforms": ["linux-x64"]
+		},
+		{
+			"asset": "FileName-macosx-amd64.sublime-package",
+			"platforms": ["osx-arm64"]
+		},
+		{
+			"asset": "FileName-macosx-arm64.sublime-package",
+			"platforms": ["osx-arm64"]
+		},
+		{
+			"asset": "FileName-win-amd64.sublime-package",
+			"platforms": ["windows-x64"]
+		}
+	]
+}
+```
+
+## `libraries`
+
+Note that the following documentation is not up to date
+with latest developments in Sublime Text (ST) (e.g. the Python 3.8 plugin host)
+or the Package Control 4.0 client package (PC4.0),
+and is mostly included to document legacy use cases.
+
+---
+
+These are typically compiled Python extensions that are supplementary to,
+or missing from Sublime Text.
+
+Each library must have a name, description, author, issues URL,
+and a list of releases.
+Each release needs a `version` and `url` or `base` and `tags` keys.
+The sublime_text, platforms and python_versions keys are optional
```

Comment:
```suggestion
The `sublime_text`, `platforms`, and `python_versions` keys are optional
```

Sorry I missed this before. I had a question about libraries and went looking for the answer. :sweat_smile: